src/wasm-lib/kcl/src/std/math.rs

```diff
@@ -533,3 +533,24 @@ pub async fn to_degrees(args: Args) -> Result<MemoryItem, KclError> {
 fn inner_to_degrees(num: f64) -> Result<f64, KclError> {
     Ok(num.to_degrees())
 }
+
+#[cfg(test)]
+mod tests {
+    use pretty_assertions::assert_eq;
+
+    use super::*;
+
+    #[test]
+    fn test_inner_max() {
+        let nums = vec![4.0, 5.0, 6.0];
+        let result = inner_max(nums);
+        assert_eq!(result, 6.0);
+    }
+
+    #[test]
+    fn test_inner_min() {
+        let nums = vec![4.0, 5.0, 6.0];
+        let result = inner_min(nums);
+        assert_eq!(result, 4.0);
+    }
```

Comment:
maybe not worth it but just in case something weird happened with negative numbers.
```suggestion
    #[test]
    fn test_inner_min() {
        let nums = vec![4.0, 5.0, 6.0];
        let result = inner_min(nums);
        assert_eq!(result, 4.0);
    }
        #[test]
    fn test_inner_min_with_neg() {
        let nums = vec![4.0, -5.0];
        let result = inner_min(nums);
        assert_eq!(result, -5.0);
    }
```